src/main/java/hu/oe/nik/szfmv/automatedcar/model/World.java

```diff
@@ -33,6 +43,68 @@ public void setHeight(int height) {
         return worldObjects;
     }
 
+    public List<WorldObject> getDynamics() {
+        List<WorldObject> dynamicObjectList = new ArrayList<>();
+        for (WorldObject item : worldObjects) {
+            if (!item.isStatic)
+                dynamicObjectList.add(item);
+        }
+        return dynamicObjectList;
+    }
+
+    public List<WorldObject> getObjectsInsideTriangle(Point a, Point b, Point c) {
+        List<WorldObject> objectInsideTriangle = new ArrayList<>();
+        for (WorldObject item : worldObjects) {
+            if (isInside(a, b, c, item))
+                objectInsideTriangle.add(item);
+        }
+        return objectInsideTriangle;
+    }
+
+    private boolean isInside(Point a, Point b, Point c, WorldObject item) {
```

Comment:
Ja jól értem írták egy függvényt, ami megnézi, hogy egy worldObject x, y koordinátája a háromszögön belül van-e.
Mit reprezentál az x,y koordinátapár? A bal felső sarkot? A tömegközéppontot? A forgatási referenciapontot? A szenzor csak akkor látja a másik autót (pl.) ha a referenciapontja/tömegközéppontja beleesik a szenzor látómezejébe?

Miért jobb ez a megoldás mint amit én ajánlottam (geometry.intersect())?